navigation/create_control/include/create_hw_interface.hpp

```diff
@@ -0,0 +1,98 @@
+#pragma once
+
+#include "create/create.h"
+
+#include <controller_manager/controller_manager.h>
+#include <hardware_interface/joint_command_interface.h>
+#include <hardware_interface/joint_state_interface.h>
+#include <hardware_interface/robot_hw.h>
+
+#include <nav_msgs/Odometry.h>
+#include <ros/ros.h>
+
+#include "create/create.h"
+
+#include <ros/ros.h>
+
+class Create2Interface : public hardware_interface::RobotHW {
+public:
+  Create2Interface(ros::NodeHandle &nh)
+      : priv_nh_(nh), model_(create::RobotModel::CREATE_2) {
+
+    hardware_interface::JointStateHandle state_handle_a(
+        "wheel_left_joint", &pos[0], &vel[0], &eff[0]);
+    jnt_state_interface.registerHandle(state_handle_a);
+
+    hardware_interface::JointStateHandle state_handle_b(
+        "wheel_right_joint", &pos[1], &vel[1], &eff[1]);
+    jnt_state_interface.registerHandle(state_handle_b);
+
+    registerInterface(&jnt_state_interface);
+
+    // connect and register the joint velocity interface
+    hardware_interface::JointHandle vel_handle_left(
+        jnt_state_interface.getHandle("wheel_left_joint"), &cmd[0]);
+    jnt_vel_interface.registerHandle(vel_handle_left);
+
+    hardware_interface::JointHandle vel_handle_right(
+        jnt_state_interface.getHandle("wheel_right_joint"), &cmd[1]);
+    jnt_vel_interface.registerHandle(vel_handle_right);
+
+    registerInterface(&jnt_vel_interface);
+    std::string dev;
+    int baud;
+    priv_nh_.param<std::string>("dev", dev, "/dev/ttyUSB0");
+    priv_nh_.param<int>("baud", baud, 115200);
+    priv_nh_.param<std::string>("tf_prefix", tf_prefix_, "create1_tf");
+
+    robot_.reset(new create::Create(model_));
+    ROS_INFO_STREAM("\n\n" << dev << baud);
```

Comment:
Use a proper log indication. You can take it from the other node.

Reply:
Deleted, this is not needed, and was only added for debugging purposes